### PR TITLE
Add new pairs on Aave v3 mainnet (Borrow & multiply)

### PR DESCRIPTION
**Pull request by piotrwitek**
All the pairs should have Borrow & multiply enabled
Only pairs with usdc, usdt, and dai as borrowed token should have stop loss automation enabled

cbETH_supply | GHO_borrow
cbETH_supply | USDC_borrow
DAI_supply | MKR_borrow
LDO_supply | USDT_borrow
LINK_supply | DAI_borrow
LINK_supply | GHO_borrow
LINK_supply | USDC_borrow
LINK_supply | USDT_borrow
LINK_supply | WETH_borrow
MKR_supply | DAI_borrow
rETH_supply | GHO_borrow
rETH_supply | USDT_borrow
sDAI_supply | GHO_borrow
sDAI_supply | USDT_borrow
sDAI_supply | WBTC_borrow
sDAI_supply | WETH_borrow
USDC_supply | GHO_borrow
USDC_supply | USDT_borrow
USDC_supply | wstETH_borrow
USDT_supply | WETH_borrow
WBTC_supply | GHO_borrow
WBTC_supply | LUSD_borrow
WBTC_supply | USDT_borrow
WBTC_supply | WETH_borrow
WETH_supply | GHO_borrow
WETH_supply | USDT_borrow
WETH_supply | WBTC_borrow
wstETH_supply | cbETH_borrow
wstETH_supply | GHO_borrow
wstETH_supply | LUSD_borrow
wstETH_supply | RPL_borrow

### CODE REVIEW

**Comment by piotrwitek** [comment]
@marcinciarka just missing icons for LDO and RPL, any idea how to get them?

File: blockchain/token-metadata-list/token-configs.ts [hunk 1]
```diff
@@ -678,6 +679,76 @@ export const tokenConfigs: TokenConfig[] = [
...
+    iconCircle: deprecated_icon,
+    coinGeckoTicker: 'rocket-pool',
+    tags: [],
+  },
+  {
+    symbol: 'CRV',
+    precision: 18,
+    digits: 5,
+    name: 'Curve',
+    icon: deprecated_icon,
```
Comment (marcinciarka) [inline blockchain/token-metadata-list/token-configs.ts:697]: We need actual icons here.

**Review by marcinciarka — CHANGES_REQUESTED** [review]

File: theme/icons/susd_circle.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+export const susd_circle = {
+  path: (
+    <svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (marcinciarka) [inline theme/icons/susd_circle.tsx:4]: Thats not needed, it's being wrapped in `svg` in the component.

**Review by marcinciarka — CHANGES_REQUESTED** [review]
MKR icon is too big, we need to change it.

**Review by marcinciarka — APPROVED** [review]

**Pull request merged**